easybuild/easyconfigs/c/Coot/Coot-0.9.8.92-binary-Linux-x86_64-scientific-linux-7.6-python-gtk2.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum - University of Basel
+# 0.9.8.92 update by Adam Huffman, University of Oxford
+
+easyblock = "Tarball"
+
+name = 'Coot'
+version = '0.9.8.92'
+versionsuffix = '-binary-Linux-x86_64-scientific-linux-7.6-python-gtk2'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot'
+description = """Coot is for macromolecular model building, model completion 
+and validation, particularly suitable for protein modelling using X-ray data."""
+
+toolchain = SYSTEM
+
+source_urls = ['http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot/binaries/release']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['3babef48f5cea17c702347e41c51e7492192c685ef1c7d8b796569eb9f80d8d9']
+
+sanity_check_paths = {
+    'files': ["bin/coot"],
+    'dirs': ["bin"]
```

Comment:
nitpicking but it doesn't make sense to check for the `bin` dir when we're already checking for `bin/coot`, maybe change this to `lib`?

Reply:
Yes, made that change now. I just inherited what was there before.